app/scripts/controllers/overview.js

```diff
@@ -440,7 +440,7 @@ angular.module('overview', ['stats', 'checks', 'ngSanitize', 'events', 'filters_
       return 'SELECT CURRENT_TIMESTAMP AS last_timestamp, ' +
         '(ended / 10000) * 10000 + 5000 AS ended_time, ' +
         'COUNT(*) / 10.0 AS qps, ' +
-        'AVG(ended - started) AS duration, ' +
+        'AVG(ended::bigint - started::bigint) AS duration, ' +
         'UPPER(regexp_matches(stmt,\'^\\s*(\\w+).*\')[1]) AS query_type ' +
         'FROM sys.jobs_log ' +
         'WHERE ended > now() - (\'15 minutes\')::interval ' +
```

Comment:
> Did you verify there is no other subtraction of timestamps used?

What about this spot?

```sql
WHERE ended > now() - (\'15 minutes\')::interval
```

Reply:
this shouldn't be a problem it's `timestamp - interval` which returns `timestamp`